lib/grape/router/route.rb

```diff
@@ -64,7 +64,8 @@ def route_path
       end
 
       def initialize(method, pattern, **options)
-        upcased_method = Grape::Http::Headers.find_supported_method(method.to_s)
+        str_method = method.to_s
```

Comment:
Minor, maybe `method_s` and `method_upcase` is a better name/model for these variables. Don't care enough.